blueprints/app/files/tests/helpers/start-app.js

```diff
@@ -2,11 +2,13 @@ import Ember from 'ember';
 import Application from '../../app';
 import config from '../../config/environment';
 
+const assign = Ember.assign || Ember.merge;
+
 export default function startApp(attrs) {
   let application;
 
-  let attributes = Ember.merge({}, config.APP);
-  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
+  let attributes = assign({}, config.APP);
+  attributes = assign(attributes, attrs); // use defaults, but you can override;
```

Comment:
We likely don't need to reassign `attributes` since `Ember.merge` and `Ember.assign` mutate the source object, `attributes`, but I kept that intact